uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-custom-work-pattern-modal.component.js

```diff
@@ -109,15 +110,26 @@ define([
     /**
      * Loads the Leave and Absences Work Pattern Change Reasons
      *
+     * @param  {Boolean} cache if to cache results of the API call, cache by default
      * @return {Promise} resolves with {Array}
      */
-    function loadJobContractRevisionChangeReasons () {
-      return OptionGroup.valuesOf('hrleaveandabsences_work_pattern_change_reason')
+    function loadJobContractRevisionChangeReasons (cache) {
+      return OptionGroup.valuesOf('hrleaveandabsences_work_pattern_change_reason', cache)
         .then(function (reasons) {
           vm.changeReasons = reasons;
         });
     }
 
+    /**
+     * Opens the work pattern change reasons for editing
+     */
+    function openWorkPatternChangeReason () {
```

Comment:
The name of the func tells it opens a change reason, while it opens an editor. Shouldn't it be "openWorkPatternChangeReasonEditor" ?